app/controllers/grade_entry_forms_controller.rb

```diff
@@ -38,6 +38,13 @@ def update
     GradeEntryStudent.refresh_total_grades(@grade_entry_form.grade_entry_students.ids)
   end
 
+  def summary
```

Comment:
My second comment is that overall I'd like to see the statistics displayed in a tab alongside the existing grades table; this is the direct analogue to the Assignment summary tab.

So to be clear, the "Grades" view can have a "Grades" tab and "Summary Statistics" tab